### PR TITLE
feat(worker): make package stage optional

### PR DESCRIPTION
**Pull request by zhujian7**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @your-reviewer

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: docs/quick-start.md [hunk 1]
```diff
@@ -64,16 +64,16 @@
  ## Create a pipeline
 
  Pipeline is a workflow that consists of a series of CI/CD stages. Cyclone supports `5` stages, among which
- two of them are mandatory:
+ one of them are mandatory:
```
Comment (supereagle) [inline docs/quick-start.md]: s/are/is/

Comment (zhujian7) [reply docs/quick-start.md]: It has been fixed

**Pull request merged**